### DISCUSSION TITLE
type="module" is a no-go with a local app

### DISCUSSION THREAD

**Discussion by sergejostir** [parcel-bundler/parcel#7205] [Q&A]
Hello.

I was using Parcel v1 to build an app that is ran locally (without a webserver). Parcel v2 forces usage for type="module" in the script tags. Browsers do a strict MIME type checking for modules, so that means that you need a webserver to serve fiels with corrent MIME type.

Is there a way around this with Parcel v2? I see that when I run "parcel serve" a non-module script tag is added, so is there a way to retain that when using build command as well?

**Comment by tedbyron** [comment]
@sergiz have you find a solution for this yet? The only way around this I've found is to manually change the output html - removing the script element with `type="module"` and on the fallback script remove `nomodule`.

Normally I get `Failed to load module script: The server responded with a non-JavaScript MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.`

opened #7583 for this

**Reply by sergejostir** [reply]
My solution is to stick with v1 😅

**Reply by tedbyron** [reply]
😦
